router.go

```diff
@@ -375,6 +387,56 @@ func (app *App) register(methods []string, pathRaw string, group *Group, handler
 	}
 }
 
+func (app *App) routeExists(method string, pathRaw string) bool {
+	return slices.ContainsFunc(app.stack[app.methodInt(method)], func(r *Route) bool {
+		if r.path == pathRaw {
+			return true
+		}
+		return false
+	})
+}
+
+// RemoveRoute is used to remove a route from the stack
+// TODO should Remove Route only be called dynamically? Can you use it to remove the /define route, leaving the app without
+// any thing on the stack but the GET /hello Route? prefix /he?
+func (app *App) RemoveRoute(path string, methods ...string) {
+	app.mutex.Lock()
+	defer app.mutex.Unlock()
+	app.removeRoute(path, methods)
+}
+
+func (app *App) removeRoute(path string, methods []string) {
+	for _, method := range methods {
+		// Uppercase HTTP methods
+		method = utils.ToUpper(method)
+
+		// Get unique HTTP method identifier
+		m := app.methodInt(method)
+		if m == -1 {
+			continue // Skip invalid HTTP methods
+		}
+
+		// Find the index of the route to remove
+		index := slices.IndexFunc(app.stack[m], func(r *Route) bool {
+			return r.Path == path
```

Comment:
Can we also add another method called `RemoveRouteByName` to remove routes by their names?

Reply:
RemoveRouteByName would remove all routes with the given name regardless of their path correct?
For instance if removing route named "test" would the both following routes be removed?
/api/test
/test

Reply:
I see now that a Route has a Name :)

Reply:
@efectn Just added the new RemoveRouteByName method in my latest commit.